README.md

```diff
@@ -1 +1,16 @@
+# Postgres backup and restore docker images
+
+## Image origin
+
 FORKED FROM: https://github.com/avencera/postgres-backup-restore that was FORKED FROM: https://github.com/schickling/dockerfiles/
+
+## Usage in the PDS
+
+Remember that this is a public repo.
+We are using only `backup` image in our infrastructure. The `restore` image is supposed to be
+used only in extraordinary situation.
+In order to get a new `backup` image to the PDS infrastructure, you have to
+locally build a docker image and push it to a `postgres-backup-restore` 
```

Comment:
Again, be specific. We have `make docker-build` and `make docker-push`.

Reply:
Well, unfortunately not in this repo. The ideal thing would be to have makefile + ci here, but I wanted to do just a quick fix.

Reply:
There is a makefile, just not in the root repo.

Reply:
Thanks, fix here: https://github.com/portworx/postgres-backup-restore/pull/6